models/project/project.go

```diff
@@ -115,6 +199,25 @@ func (u *ProjectStruct) Lists() []*list.ListStruct {
 	return slice
 }
 
+func (u *ProjectStruct) NoneList() *list.ListStruct {
```

Comment:
__[gometalinter]__
* [gotype] undeclared name: list
* [golint] exported method ProjectStruct.NoneList should have comment or be unexported